core/domain/html_validation_service.py

```diff
@@ -807,7 +807,8 @@ def get_filename_with_dimensions(old_filename, exp_id):
     file_system_class = (
         fs_domain.ExplorationFileSystem if feconf.DEV_MODE
         else fs_domain.GcsFileSystem)
-    fs = fs_domain.AbstractFileSystem(file_system_class(exp_id))
+    fs = fs_domain.AbstractFileSystem(file_system_class(
```

Comment:
Why the change in this file and the next one?

Reply:
So that it supports the new system of storage. We had exp_id for the migration in production. Since its over we can change it now.

Reply:
The other file is the test file of this, so that too includes the same change